web/src/popover_menus.js

```diff
@@ -983,59 +987,105 @@ export function initialize() {
                 ),
             );
         },
+        onMount(instance) {
+            starred_messages_on_mount(instance);
+        },
         onHidden(instance) {
             instance.destroy();
             popover_instances.starred_messages = undefined;
         },
     });
 
+    function drafts_sidebar_on_mount(instance) {
+        const $popper = $(instance.popper);
+        $popper.addClass("drafts-popover");
+        popover_instances.drafts = instance;
+
+        $popper.one("click", "#delete_all_drafts_sidebar", () => {
+            drafts.confirm_delete_all_drafts();
+            instance.hide();
+        });
+    }
+
     // Drafts popover
     register_popover_menu(".drafts-sidebar-menu-icon", {
         ...left_sidebar_tippy_options,
-        onMount(instance) {
-            const $popper = $(instance.popper);
-            $popper.addClass("drafts-popover");
-            popover_instances.drafts = instance;
-
-            $popper.one("click", "#delete_all_drafts_sidebar", () => {
-                drafts.confirm_delete_all_drafts();
-                instance.hide();
-            });
-        },
         onShow(instance) {
             popovers.hide_all_except_sidebars();
 
             instance.setContent(parse_html(render_drafts_sidebar_actions({})));
         },
+        onMount(instance) {
+            drafts_sidebar_on_mount(instance);
+        },
         onHidden(instance) {
             instance.destroy();
             popover_instances.drafts = undefined;
         },
     });
 
+    function all_messages_on_mount(instance) {
```

Comment:
I'd kind of like to see all of the `_on_mount` functions defined together, and then keep the various `register_` functions grouped, too. I think that'll be easier for future contributors to scan.